pkg/gzip/file.go

```diff
@@ -18,6 +19,16 @@ type File struct {
 	Path    string
 }
 
+// Compression and Decompression functions used by the File.Process method.
+// The implementation depends on the build tags tinygo and !tinygo.
```

Comment:
"The implementation depends" comment may not be needed. Build tag dependencies are a given and change over time. IOW in the future maybe some other build tags would come into use, such that this comment would not be quite accurate. 
But I understand it's a good idea to let people know this happens. I don't know, wdyt of slightly different wording. 
"Bare metal support is enabled by, for example, the tinygo build tags"
WDYT?

Reply:
What do you think of the new proposal I pushed?